R/parameters.R

```diff
@@ -376,6 +376,17 @@ sircovid_date <- function(date) {
   days_into_2020
 }
 
+# Converts dates from from sircovid_Date as used in the MCMC to a Date
+# Automatically converts type
+sircovid_date_to_date <- function(sircovid_date)
```

Comment:
This should have a more informative name, perhaps `sircovid_date_as_Date()` to make the class of the return type clearer

Reply:
Also there are a two places in the plots where this same transform is used, I'd suggest switching these out to this function too:
- https://github.com/mrc-ide/sircovid/blob/master/R/sampler.R#L210
- https://github.com/mrc-ide/sircovid/blob/master/R/pmcmc.R#L644